packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -1399,7 +1399,7 @@ function updateHostRoot(current, workInProgress, renderLanes) {
     } else {
       // The outermost shell has not hydrated yet. Start hydrating.
       enterHydrationState(workInProgress);
-      if (enableUseMutableSource && supportsHydration) {
```

Comment:
The variable `supportsHydration` has been judged on [line 1354](https://github.com/zh-lx/react/blob/main/packages/react-reconciler/src/ReactFiberBeginWork.new.js#L1354), when the code executes to [line 1402](https://github.com/zh-lx/react/blob/main/packages/react-reconciler/src/ReactFiberBeginWork.new.js#L1402), the value of `supportsHydration` must be `true`. So it doesn't need to be judged again.